app/controllers/graphql_controller.rb

```diff
@@ -9,8 +9,7 @@ def execute
     query = params[:query]
     operation_name = params[:operationName]
     context = {
-      # Query context goes here, for example:
-      # current_user: current_user,
+      current_user: current_user,
```

Comment:
do we also need to force authorisation? what happens when you run graphql when you are NOT logged in? should we look at authenticating via JWT? as per example something like https://github.com/saramic/real-code-runner/commit/9af6752aa4d15ff59a9153cb24bdf91adb49a71b